src/main/kotlin/no/nav/syfo/client/LegeSuspensjonClient.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.syfo.client
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.util.*
+import no.nav.syfo.VaultSecrets
+import no.nav.syfo.log
+import java.io.IOException
+
+@KtorExperimentalAPI
+class LegeSuspensjonClient(
+    private val endpointUrl: String,
+    private val secrets: VaultSecrets,
+    private val stsClient: StsOidcClient,
+    private val httpClient: HttpClient
+) {
+
+    suspend fun checkTherapist(therapistId: String, ediloggid: String, oppslagsdato: String): Suspendert {
+        val httpStatement = httpClient.get<HttpStatement>("$endpointUrl/api/v1/suspensjon/status") {
```

Comment:
Vi fjernet retry fra denne i padm2regler for å slippe dobbel retry, men nå som denne metoden er her kan vi kanskje legge det på igjen?

Reply:
Retry it is!